.github/workflows/push-important-models.yml

```diff
@@ -49,15 +49,17 @@ jobs:
                 fi
             done
             matrix_string=$(printf '"%s", ' "${model_arrays[@]}" | sed 's/, $//')
+            echo $matrix_string
             echo "matrix=[$matrix_string]" >> $GITHUB_OUTPUT
   test_modified_files:
     needs: get_modified_models
     name: Slow & FA2 tests
+    runs-on: ubuntu-latest
     runs-on: [single-gpu, nvidia-gpu, a10, ci]
     container:
       image: huggingface/transformers-all-latest-gpu
       options: --gpus all --privileged --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
-    if: ${{ needs.get_modified_models.outputs.matrix != '[]' && needs.get_modified_models.outputs.matrix != '' }}
+    if: ${{ needs.get_modified_models.outputs.matrix != '[]' && needs.get_modified_models.outputs.matrix != '' && fromJson(needs.get_modified_models.outputs.matrix)[0] != null }}
```

Comment:
That's the most efficient condition i could use, simply converting to an array and trying to access to the first element seemed to work

Reply:
Are we always going to be able to safely index on `0` here?

Reply:
Great - thanks for confirming!